src/main/resources/public/js/pages/profile.js

```diff
@@ -353,6 +426,24 @@ class PasswordAccount extends React.Component {
     }
 }
 
+const ConditionalClaimsField = ({ children, voluntaryClaims = [], essentialClaims = [], field }) =>
+    ((!voluntaryClaims.length && !essentialClaims.length) || voluntaryClaims.includes(field) || essentialClaims.includes(field)) &&
+        children;
+
+const ConditionalClaimsForm = ({ children, voluntaryClaims = [], essentialClaims = [], fields }) =>
+    ((!voluntaryClaims.length && !essentialClaims.length) || fields.some(item => voluntaryClaims.includes(item) || essentialClaims.includes(item))) &&
+        children;
+
+const conditionalClaimsRequired = (field, defaultRequired, essentialClaims = []) =>
+    (!!essentialClaims.length && essentialClaims.includes(field)) || defaultRequired
+
+const getConditionalClaims = (searchLocation) => {
+    const searchParams = new URLSearchParams(searchLocation);
```

Comment:
This won't work in IE (without a polyfill): https://caniuse.com/#feat=urlsearchparams; it's easy to get rid of it though.

Reply:
I think a polyfill has been added (cf https://github.com/ozwillo/ozwillo-portal/pull/422/files#diff-df93958ada5f78e6435f874ba400ec41R8) but I don't see it added in package.json

Reply:
Yes I added a polyfill. 
I didn't realize it wasn't added to the package.json when I installed it (because I already had it on my computer apparently). I will `--save` it in a new commit.